frontend/tests/unittests/pages/FormStaffPage.spec.ts

```diff
@@ -2,6 +2,10 @@ import { createRouter, createMemoryHistory } from 'vue-router';
 import { describe, it, expect } from "vitest";
 import { mount } from '@vue/test-utils';
 import FormStaffPage from '@/pages/FormStaffPage.vue';
+import ForestClientUserSession from '../../../src/helpers/ForestClientUserSession';
+
+
+const isAdminInd = ["CLIENT_ADMIN"].some(authority => ForestClientUserSession.authorities.includes(authority));
```

Comment:
Even though this can work during tests, it should be handled differently. 

How it is coded here implies that a user session exists and someone is logged in. A unit test usually contains a more limited scope. 

It would be better to have a test that will mock the behaviour of the user session, by setting a session for an admin and another test with a session for an editor.